app/assets/stylesheets/modules/overlay.css.scss

```diff
@@ -4,7 +4,11 @@
   top: 0;
   min-width: 100%;
   height: 100%;
-  z-index: 3;
+  z-index: 5;
   background-color: white;
   opacity: 0.6;
+
+  &.overlay--indoor {
+    z-index: 3; // session cards and info are on top of overlay
```

Comment:
I see that we have some others `z-index`es. Would it make sense to extract them as vars?

Reply:
good point, I'll keep that in mind